mappings/net/minecraft/block/piston/PistonHandler.mapping

```diff
@@ -1,15 +1,25 @@
 CLASS net/minecraft/class_2674 net/minecraft/block/piston/PistonHandler
-	FIELD field_12243 direction Lnet/minecraft/class_2350;
+	FIELD field_12243 motionDirection Lnet/minecraft/class_2350;
 	FIELD field_12244 posTo Lnet/minecraft/class_2338;
 	FIELD field_12245 movedBlocks Ljava/util/List;
 	FIELD field_12246 brokenBlocks Ljava/util/List;
+	FIELD field_12247 retracted Z
+	FIELD field_12248 pistonDirection Lnet/minecraft/class_2350;
 	FIELD field_12249 world Lnet/minecraft/class_1937;
 	FIELD field_12250 posFrom Lnet/minecraft/class_2338;
 	METHOD <init> (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;Z)V
 		ARG 1 world
 		ARG 2 pos
 		ARG 3 dir
+		ARG 4 retracted
 	METHOD method_11536 getBrokenBlocks ()Ljava/util/List;
 	METHOD method_11537 calculatePush ()Z
+	METHOD method_11538 canMoveAdjacentBlock (Lnet/minecraft/class_2338;)Z
+		ARG 1 pos
+	METHOD method_11539 setMovedBlocks (II)V
```

Comment:
Mind map the int params? that would be helpful

Reply:
done